Makefile

```diff
@@ -39,6 +39,10 @@ build-all:
 aro: generate
 	go build -tags aro,containers_image_openpgp,codec.safe -ldflags "-X github.com/Azure/ARO-RP/pkg/util/version.GitCommit=$(VERSION)" ./cmd/aro
```

Comment:
Then, here we could have an `if` statement in `make aro` that runs the tag check if `RELEASE`/IS_OFFICIAL_RELEASE is true. Otherwise it uses `VERSION`

Reply:
That makes more sense, I like it. I'll give it a shot and update this PR.